Readme.md

```diff
@@ -50,26 +50,34 @@ Explorer for Endevor is included with Eclipse Che version 7.6.0 and above. Check
 
 Create a Host in Explorer for Endevor to enable it to access the repository you want to explore.
 
-### **Create a Host**
+### **Create a Profile**
 
-Create a Host in Explorer for Endevor to specify the data source you want to work with. This allows you to view and explore the selected Endevor repositories.
+Create a Profile in Explorer for Endevor to specify the data source that you want to work with. This allows you to view and explore the selected Endevor repositories.
 
+**Follow these steps:**
+
+1. Click on the + icon.
+2. Enter a name for your Profile.
+3. Enter your Endevor URL in the format `https://host:port`.
+4. (Optional) To add your mainframe credentials to your profile, enter your username and password. Adding your credentials to your profile lets you access different data sources without entering your credentials.
+5. Set profile settings to True or False to indicate that your profile should Reject or Accept connections with self-signed certificates.  
+Your Profile is now available in the panel on the left.
 
-![](https://github.com/eclipse/che-che4z-explorer-for-endevor/blob/v0.9.1/create_host.gif)
+6. Click on the profile in the panel.  
```

Comment:
The bottom four steps are not part of the procedure to "create a profile". 6 and 7 are a new procedure to add a data source, which can be explained in a sentence as it's fewer than 3 steps. 

Info on how to delete profiles could be condensed into a single sentence too and less prescriptively - just say "to delete a profile, do this".

I would put these under another subheading called "working with profiles" or something.